### ISSUE TITLE
How to pass proxy capabilities with .side.yml file for selenium-side-runner

### ISSUE THREAD

**Issue by Volley97** [SeleniumHQ/selenium-ide#484]
## 🐛 Bug Report

Here is .side.yml file content

```сapabilities:
browserName: "chrome"
    proxy:
            proxyType: "MANUAL"
            http: "http://localhost:5555"
baseUrl: "URL"
```

when you start the test opens in normal mode, not through a proxy. In the settings, nothing changes. What should be added or changed in the file?

selenium-side-runner --version
3.4.2
Linux Mint Mate 19

**Comment by Yrds** [comment]
Could you format the .side.yml content to better readability?  

The answer:

You are using http capability, maybe if you try httpProxy instead, you can set the right proxy url.

See more in: https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#proxy-json-object

**Comment by Volley97** [comment]
If I make the same syntax:
```
capabilities:
  browserName: "firefox"
baseUrl: "https://icompas.thome.com.sg"
server: "http://localhost:5555"
```
Mistake:
```
Test suite failed to run

    WebDriverError: No Matching Request
    Incoming Request Method: POST
    Incoming Request URI: https://localhost:5555/session
    Incoming Request Headers: {Accept=[application/json; charset=utf-8], Content-Length=[49], Content-Type=[application/json;charset=UTF-8], Host=[localhost:5555], Connection=[close]}
    Incoming Request Body: {"desiredCapabilities":{"browserName":"firefox"}}
```

Another syntax:
```

capabilities:
browserName: "firefox"
 proxy:
     proxyType: "MANUAL"
     httpProxy: "http://localhost:5555"
baseUrl: "https://icompas.thome.com.sg"


```
It`d opened site, but not in proxy

**Comment by Yrds** [comment]
With a squid server hosted on my machine i could reproduce the bug:

.side.yml:

```
capabilities:
 browserName: "firefox"
 proxy:
     proxyType: "manual"
     httpProxy: "http://localhost:443"
```

The browser is initializing and running the test normally, but without proxy

**Comment by ekondrashev** [comment]
@Yrds so, correct me if I understood it wrong, but are you saying that if proxy will be hosted different then on `localhost` it should work ok?

**Comment by tourdedave** [comment]
The runner doesn't currently support proxy capabilities. Marking as a feature request.

**Comment by corevo** [comment]
Added proxy capabilities to the runner, [docs here](https://github.com/SeleniumHQ/selenium-ide/blob/master/packages/selenium-side-runner/README.md#using-a-proxy-server).  
Will be available in 3.5.0
